lib/grammers-client/src/types/update.rs

```diff
@@ -15,6 +15,8 @@ use std::sync::Arc;
 pub enum Update {
     /// Occurs whenever a new text message or a message with media is produced.
     NewMessage(Message),
+    /// Occurs when a message is updated.
+    MessageUpdated(Message),
```

Comment:
I think we should caml this `MessageEdited`, as per https://t.me/gramme_rs/11222.

Reply:
Changed

Reply:
> I think we should caml this `MessageEdited`, as per [t.me/gramme_rs/11222](https://t.me/gramme_rs/11222).

Changed